files/etc/ublue-update.d/check/00-system-update-check.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
I think we should implement this one in python

rpm-ostree has a —json flag and skopeo outputs Jain so this is actually easier to implement in python

Reply:
I can look into writing that